src/bp-core/bp-core-update.php

```diff
@@ -487,6 +487,10 @@ function bp_version_updater() {
 			bb_update_to_2_5_80();
 		}
 
+		if ( $raw_db_version < 21091 ) {
```

Comment:
@bb-yudhisthir 
Need to update db version in this file `buddyboss-platform/src/class-buddypress.php`.
`$this->db_version = 21091;` 
Please check latest db version with release.